reference-implementation/to-upstream-wpts/writable-streams/general.js

```diff
@@ -80,3 +80,22 @@ test(() => {
     }
   );
 }, 'ws.getWriter() on an errored WritableStream');
+
+promise_test(() => {
+  const ws = new WritableStream({});
+
+  const writer = ws.getWriter();
+  writer.releaseLock();
+
+  return writer.closed.then(
```

Comment:
Yeah, but then I would have to write about this much code to test that they're equal (line 96), so I'm not sure it gains much :-/.